spec/Functions.tex

```diff
@@ -734,13 +734,6 @@ \subsection{The Ref Return Intent}
 behavior can be useful for adjusting the behavior depending on the
 calling context.
 
-\begin{craychapel}
-Previous versions of Chapel included an implicit \chpl{setter} param of
-type \chpl{bool} for \chpl{ref} return intent functions. The old behavior
-can be recreated by making both ref-pair functions call another function
-and explicitly pass the \chpl{setter} value, as in the the example below.
-\end{craychapel}
-
```

Comment:
It's been updated at least. I'm not sure if the spec should call these "getter-setter pairs" or "ref-pairs". And I'm not sure if the example in the spec around line 772 is any good.